### PR TITLE
[#208] fix storing JSON into DB

### PR DESCRIPTION
**Pull request by cupakob**
## Motivation and context

bugfix issue #208 

## Testing

unit tests added

### CODE REVIEW

File: backend/contact/database/datatypes_test.go [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+    // given
+    jsonb := &database.JSONB{}
+    _ = jsonb.Scan(payload)
+
+    // when
+    value, err := jsonb.Value()
+
+    // then
+    assert.NoError(t, err)
+    assert.NotNil(t, value)
```
Comment (kachar) [inline backend/contact/database/datatypes_test.go:85]: @cupakob Maybe we can add a test to make sure the output is the same as the input payload

```suggestion
    assert.NotNil(t, value)
    assert.JSONEqf(t, payload, fmt.Sprintf("%v", value), "payload-differs")
```

**Comment by kachar** [comment]
@cupakob I've tested this PR locally, it does the convertion to string properly, tests coonfim that, but the actual pain of not storing the data in the table still persists.

Even with this fix it seems that we're missing something else to complete the issue

Do you have any idea what else might cause it ?

File: backend/contact/database/datatypes_test.go (hunk 1 above)
Comment (cupakob) [reply backend/contact/database/datatypes_test.go]: then `assert.NotNil` is not needed anymore. I've never tested, if the entry is correct persisted. I just read the comment, that the marshaler is not doing it's job properly. And that was my approach to fix the problem.

**Comment by cupakob** [comment]
I've never tested, if the entry is correct persisted. I just read the comment, that the marshaler is not doing it's job properly. And that was my approach to fix the problem.

**Comment by kachar** [comment]
@cupakob Do you think it will be more appropriate to merge & close this one and open a new issue?

**Comment by cupakob** [comment]
yes, i think, it will be better to create another one, becaus the problem from the description is solved.

**Pull request merged**